src/core/platforms/ios/iosplatformutilities.mm

```diff
@@ -35,6 +36,28 @@
 
 IosPlatformUtilities::IosPlatformUtilities() : PlatformUtilities() {}
 
+@interface DocViewController
```

Comment:
this part of the code was in a separate header in the demo code. Any reason behind this?

Reply:
@3nids , not really other than I thought it was nifty to keep it within platform utilities as it's ever only used there. Got mild feelings about it, can easily be changed if you have stronger feelings :)

Thanks for the review.

Reply:
nah, keep it simple and move forward 👍